src/middlewared/middlewared/main.py

```diff
@@ -731,6 +737,13 @@ async def run(self, ws, request, conndata):
                 options['jail'] = data.get('jail') or options.get('jail')
                 if options.get('vm_id'):
                     options['vm_data'] = await self.middleware.call('vm.get_instance', options['vm_id'])
+                if options.get('chart_release_name'):
+                    options['chart_release'] = await self.middleware.call(
+                        'chart.release.get_instance', options['chart_release_name']
+                    )
+                    if not options.get('pod_name') or options.get('container_name'):
```

Comment:
You probably want to raise that before wasting the time of doing the get_instance call?

Reply:
The if statement also looks off, algebraically speaking